cocos/core/geometry/sphere.ts

```diff
@@ -212,8 +210,8 @@ export class Sphere {
      * Get the bounding points of this shape
      * @zh
      * 获取此形状的边界点。
-     * @param {Vec3} minPos 最小点。
-     * @param {Vec3} maxPos 最大点。
+     * @param {Vec3} minPos @en Bottom(negative axis) point of the sphere. @zh 最小点。
```

Comment:
Minimum position have minimum x, y and z on all three axis. So it's not only the bottom point, and the values on each axis don't need to be negative. So the following expression is ok:

The minimum position of the sphere

Reply:
https://www.google.com/search?q=minimum+position+aabb&biw=1507&bih=846&sxsrf=ALiCzsZGfFOJ1t_icXk7WWn6NSR6Al_4-w%3A1651828798644&ei=Puh0YvCDJ86goATp4LmQBA&ved=0ahUKEwjwmpfmxcr3AhVOEIgKHWlwDkI4ChDh1QMIDg&oq=minimum+position+aabb&gs_lcp=Cgdnd3Mtd2l6EAxKBAhBGABKBAhGGABQAFgAYABoAHABeACAAQCIAQCSAQCYAQA&sclient=gws-wiz
can't find such expression.

Reply:
considering not using z-up, but the quadrant which xyz both positive, and that's why commented in parentheses. same time  i'm not saying it's the correct statement， just `minimum position` don't seems like a best choice. if no other choice this could be an option.